conan/api/subapi/config.py

```diff
@@ -191,3 +191,21 @@ def appending_recursive_dict_update(d, u):
             appending_recursive_dict_update(settings, settings_user)
 
         return Settings(settings)
+
+    def clean(self):
```

Comment:
Does it make sense that this command automatically calls ``reinit()``, as it destroys files that are cached by the api?